src/main/webapp/app/components/util/secured-image.component.ts

```diff
@@ -22,6 +24,11 @@ export class SecuredImageComponent implements OnChanges {
     @Input()
     private src: string;
     private src$ = new BehaviorSubject(this.src);
+    private value = 0;
+    private retryCounter = 0;
+
+    @Output()
+    endLoadingProcess = new EventEmitter();
```

Comment:
Please Type this EventEmitter. Maybe add a Enum for the Possible Return Values?

Reply:
ok - created an enum